src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -225,6 +228,7 @@ export default class ReviewCollapsibleChapter extends React.Component {
                         }}
                         buttonText="Update page"
                         buttonClass="usa-button-primary"
+                        aria-label={ariaLabel}
```

Comment:
This seems fine as long as the prop is camel cased, just because that's the "React way"